pubpub/platform
# fix: make member  id selector behave more as expected

- **fix: restore pubfields and tokens for action run form that were accidentally deleted in #399**
- **chore: cleanup the query params for the member add field a bit**
- **fix: prevent the member add component from continually rerendering the page when focus is lost**

## Issue(s) Resolved
#478 

## Test Plan

1. In croccroc, setup an email action on a stage
2. Configure it to have the recepient be all@pubpub.org by default, and to use the pubfield of `croccroc:member-id`, as well as the other things.
3. Set up a rule to run that action once a new pub enters a stage
4. Lookup the memberid for a user on croccroc that is not `all@pubpub.org` 
5. Create a pub in the stage you setup the email action for, and enter the memberId you found from step 4. (alternatively pull in https://github.com/pubpub/v7/pull/499 instead and just use the memberselect component instead)
6. Go to inbucket and observe that the email is sent to the member from the PubField, not `all@pubpub.org`
7. Manually run the action with the same settings, and observe the same.
8. Create another pub, but this time do not set a memberId
9. Observe that no email is sent, and the action log should have an error

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
